var/spack/repos/builtin/packages/embree/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+class Embree(CMakePackage):
+    """High-performance ray tracing kernel"""
+
+    homepage = "https://embree.github.io/"
+    url = "https://github.com/embree/embree/archive/v3.5.1.tar.gz"
+    generator = 'Ninja'
+
+    version('3.5.1', sha256='d6c3f00722f647c2f2c8b5c78aa506b118d486e7ff6baafb9158f06ad65d736e')
+
+    depends_on('ispc', type='build')
+    depends_on('ninja', type='build')
```

Comment:
Do we not need to depend on CMake/constrain its version?

Reply:
This is not strictly needed because CMakePackage already depends on cmake, but I added the minimum version to the package (Except VRPN because they don't state a minimum version).